RecoVertex/AdaptiveVertexFinder/python/inclusiveNegativeVertexing_cff.py

```diff
@@ -0,0 +1,62 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoVertex.AdaptiveVertexFinder.inclusiveCandidateVertexFinder_cfi import *
+from RecoVertex.AdaptiveVertexFinder.candidateVertexMerger_cfi import *
+from RecoVertex.AdaptiveVertexFinder.candidateVertexArbitrator_cfi import *
+
+
+inclusiveCandidateNegativeVertexFinder = inclusiveCandidateVertexFinder.clone(
+   vertexMinAngleCosine = cms.double(-0.95) 
```

Comment:
here and in other ```x = y.clone( par = ``` cases please drop the type specifications on the right hand side for changes in existing parameter values.
This is a more safe way against syntax mistakes

Reply:
Done in https://github.com/cms-sw/cmssw/pull/23206/commits/4e37f178d6a3ee3b5778ba933e82064eaf694aa4

I did the same changes in inclusiveVertexing_cff.py in https://github.com/cms-sw/cmssw/pull/23206/commits/4666d808010a3b25a37ebb86fec277bb3e0e529e

And the same changes in RecoBTag/SecondaryVertex/python/pfInclusiveSecondaryVertexFinderNegativeTagInfos_cfi.py with https://github.com/cms-sw/cmssw/pull/23206/commits/575defa79289ec3f2b9bd27f3767e3defa0f3a08

btw do you know if there is a better syntax for https://github.com/cms-sw/cmssw/pull/23206/commits/4e37f178d6a3ee3b5778ba933e82064eaf694aa4#diff-8286bd0001c5e9e542e42a3b99496ad5R10 . It seems a bit clunky to call clone inside the clone, but if you do cms.PSet it resets to default values rather than cloning.

Reply:
@emilbols You can do
```python
inclusiveCandidateNegativeVertexFinder = inclusiveCandidateVertexFinder.clone(
   vertexMinAngleCosine = -0.95, 
   clusterizer = dict(
        clusterMinAngleCosine = -0.5
        )
)
```

and
```python
pfDeepCSVNegativeTagInfos = pfDeepCSVTagInfos.clone(
	computer = dict(
			trackSelection = dict(
				sip3dSigMax = 10.0
			)
		)
	)
```

Reply:
@makortel Thanks!